### ISSUE TITLE
Akamai edgedns is not recognized

### ISSUE THREAD

**Issue by dejeroWilliamScott** [go-acme/lego#1224]
Following the example provided on https://go-acme.github.io/lego/dns/edgedns/, I attempted to create a new wildcard certificate.
```
/ # lego --domains="fake.com" --email="foo@fake.com" --dns="edgedns" --server=https://acme-staging-v02.api.letsencrypt.org/directory --accept-tos run
2020/07/29 20:53:07 No key found for account foo@fake.com. Generating a P384 key.
2020/07/29 20:53:07 Saved key to /.lego/accounts/acme-staging-v02.api.letsencrypt.org/foo@fake.com/keys/foo@fake.com.key
2020/07/29 20:53:07 unrecognized DNS provider: edgedns
```
I suspect this issue originates from an old build of the lego docker image.  I was able to reproduce using latest and v3.8.0, but I received a different error using the dev image (which recognized the DNS).
```
/ # lego --domains="*.fake.com" --email="foo@fake.com" --dns="edgedns" --server=https://acme-staging-v02.api.letsencrypt.org/directory --accept-tos run
2020/07/29 21:03:37 No key found for account foo@fake.com. Generating a P384 key.
2020/07/29 21:03:37 Saved key to /.lego/accounts/acme-staging-v02.api.letsencrypt.org/foo@fake.com/keys/foo@fake.com.key
2020/07/29 21:03:37 [INFO] acme: Registering account for foo@fake.com
!!!! HEADS UP !!!!

Your account credentials have been saved in your Let's Encrypt
configuration directory at "/.lego/accounts".

You should make a secure backup of this folder now. This
configuration directory will also contain certificates and
private keys obtained from Let's Encrypt so making regular
backups of this folder is ideal.
2020/07/29 21:03:38 [INFO] [*.fake.com] acme: Obtaining bundled SAN certificate
2020/07/29 21:03:38 [INFO] [*.fake.com] AuthURL: https://acme-staging-v02.api.letsencrypt.org/acme/authz-v3/85161279
2020/07/29 21:03:38 [INFO] [*.fake.com] acme: use dns-01 solver
2020/07/29 21:03:38 [INFO] [*.fake.com] acme: Preparing to solve DNS-01
2020/07/29 21:03:38 [INFO] [*.fake.com] acme: Cleaning DNS-01 challenge
2020/07/29 21:03:38 [WARN] [*.fake.com] acme: cleaning up failed: edgedns: Get "https:///config-dns/v2/zones/fake.com/names/_acme-challenge.fake.com./types/TXT": http: no Host in request URL
2020/07/29 21:03:38 [INFO] Deactivating auth: https://acme-staging-v02.api.letsencrypt.org/acme/authz-v3/85161279
2020/07/29 21:03:38 Could not obtain certificates:
        error: one or more domains had a problem:
[*.fake.com] [*.fake.com] acme: error presenting token: edgedns: Get "https:///config-dns/v2/zones/fake.com/names/_acme-challenge.fake.com./types/TXT": http: no Host in request URL
```

**Comment by ldez** [comment]
Hello, edgedns will be available in the next version v3.9.0.

**Comment by dejeroWilliamScott** [comment]
Ah, okay, that makes sense.  I'm using v3.8.0 (latest).
```
docker image ls
...
goacme/lego                                    latest              09acfd06e13f        3 weeks ago         53.9MB
goacme/lego                                    v3.8.0              09acfd06e13f        3 weeks ago         53.9MB
```
